container/attribute.go

```diff
@@ -4,75 +4,99 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/container"
 )
 
-type (
-	Attribute  container.Attribute
-	Attributes []Attribute
-)
-
-// NewAttribute creates and initializes blank Attribute.
+// Attribute represents v2-compatible container attribute.
 //
-// Defaults:
-//  - key: "";
-//  - value: "".
-func NewAttribute() *Attribute {
-	return NewAttributeFromV2(new(container.Attribute))
-}
+// Attribute is mutually compatible with github.com/nspcc-dev/neofs-api-go/v2/container.Attribute
+// message. See ReadFromV2 / WriteToV2 methods.
+//
+// Instances can be created using built-in var declaration.
+//
+// Note that direct typecast is not safe and may result in loss of compatibility:
+// 	_ = Attribute(container.Attribute{}) // not recommended
+type Attribute container.Attribute
 
-func (a *Attribute) SetKey(v string) {
-	(*container.Attribute)(a).SetKey(v)
-}
+// Attributes groups container attributes.
+type Attributes []Attribute
```

Comment:
Do we really need this type?

Reply:
think that it is good to have such since `object` has the same

also, it could be useful to convert to/from `v2` types and not do `for` loops in dapps

Reply:
I agree that it is useful, but I propose to encapsulate all slice properties in methods:
```
// _ = Attributes([]Attribute{}) // not recommended

func (x Attributes) Len() int { ... }

func (x Attributes) Iterate(func(Attribute) bool) { ... }

func (x *Attributes) Append(...Attribute) { ... }

// etc.
```

What do you think? Otherwise IMO it's easier to work with `[]Attribute` directly.

Reply:
sure, added